### PR TITLE
examples: add maxmod streaming example

### PR DESCRIPTION
**Pull request by mdigiorgio**

### CODE REVIEW

File: examples/maxmod/streaming/source/main.c [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    mm_ds_system mmSys;
+    mmSys.mod_count    = 0;
+    mmSys.samp_count   = 0;
+    mmSys.mem_bank     = 0;
+    mmSys.fifo_channel = FIFO_MAXMOD;
+    mmInit(&mmSys);
+
+    // Open the stream
+    mm_stream stream;
+    stream.sampling_rate = 25000;
```
Comment (asiekierka) [inline examples/maxmod/streaming/source/main.c:54]: Where d these values originate from?

Comment (mdigiorgio) [reply examples/maxmod/streaming/source/main.c]: I've changed the example to play a WAV track which should be more useful

File: examples/maxmod/streaming/source/main.c [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+// Audio track credits:
+//  https://www.looperman.com/loops/detail/222561/kingmike16-120bpm-techno-synth-loop
+
+#include <stdio.h>
+
+#include <filesystem.h>
+#include <maxmod9.h>
+#include <nds.h>
+
+#define WAV_FILENAME "nitro:/fstest/looperman.wav"
```
Comment (AntonioND) [inline examples/maxmod/streaming/source/main.c:17]: Just a small detail, move the wav file one level up (remove the `fstest` folder).

Comment (mdigiorgio) [reply examples/maxmod/streaming/source/main.c]: Done

**Comment by AntonioND** [comment]
Thanks!

**Pull request merged**